tensorflow/python/training/saver.py

```diff
@@ -1070,6 +1070,10 @@ def save(self, sess, save_path, global_step=None, latest_filename=None,
     if not isinstance(sess, session.SessionInterface):
       raise TypeError("'sess' must be a Session; %s" % sess)
 
+    # Note a few lines above save_path was set to os.path.dirname(save_path)
```

Comment:
I don't have a Windows machine to try, but

    os.path.exists(os.path.dirname("whatever"))

indeed returns False on my machine. A solution would be to use

    if not gfile.IsDirectory(os.path.abspath(os.path.dirname(save_path))):

on line 1312. I'll make that change.